server/comment/hooks.js

```diff
@@ -43,6 +43,9 @@ async function sendEmailNotification(comment, db) {
     author: author.profile,
     ...pick(comment, ['id', 'content', 'createdAt'])
   };
-  const recipients = map(repository.repositoryUsers, it => it.user.email);
+  const recipients = reduce(repository.repositoryUsers, (acc, it) => {
```

Comment:
Although this is faster consider using `lodash/without` to make it a bit more readable and compact:
```javascript
const collaborators = map(repository.repositoryUsers, 'user.email');
const recipients = without(collaborators, author.email);
```
Also what happens if there are none 😉